dotcom-rendering/src/web/experiments/lib/ab-exclusions.ts

```diff
@@ -0,0 +1,49 @@
+import { Participations } from '@guardian/ab-core';
+import {
+	getParticipationsFromLocalStorage,
+	setParticipationsInLocalStorage,
+} from './ab-localstorage';
+
+/**
+ * Occasionally we may want to track long-running behavior of browsers in a certain test,
+ * without adding new browsers to the test bucket. We cannot rely on the GU_mvt_id bucketing alone to do this.
+ * You can use this method to show the gate based on local storage participation key ("gu.ab.participations") instead.
+ *
+ * After the cut off date users without the local storage participation key won't see the gate at all.
+ *
+ * Caveats - There will be natural churn of users to account for. Users can manually delete localStorage at any time.
+ * Some browsers also automatically delete local storage specifically on Webkit browsers (Safari/iOS) since iOS and iPadOS 13.4, Safari 13.1 on macOS
```

Comment:
```suggestion
 * Some browsers also automatically delete local storage after some time.
 * Notably: ITP in Safari since iOS and iPadOS 13.4, Safari 13.1 on macOS @see https://webkit.org/tracking-prevention/#intelligent-tracking-prevention-itp
```